apps/system/js/sim_lock_system_dialog.js

```diff
@@ -123,34 +129,35 @@
   };
 
   SimLockSystemDialog.prototype.updateHeight = function() {
-    SystemDialog.prototype.updateHeight.apply(this, arguments);
+    var height = Service.query('LayoutManager.height');
+    this.containerElement.style.height = height + 'px';
+    // We need to manually set the gaia-dialog height,
+    // as it is fixed positioned
+    this.element.style.height = height + 'px';
+    this.debug('updateHeight: new height = ' + height);
     document.activeElement.scrollIntoView(false);
   };
 
   SimLockSystemDialog.prototype._registerEvents = function() {
-    this.dialogDone.onclick = this.verify.bind(this);
-    this.dialogSkip.onclick = this.skip.bind(this);
+    this.dialogDone.addEventListener('click', this.verify.bind(this));
+    this.dialogSkip.addEventListener('click', this.skip.bind(this));
     this.dialogDone.onmousedown = this._handle_mousedown.bind(this);
     this.dialogSkip.onmousedown = this._handle_mousedown.bind(this);
```

Comment:
If you're changing `.onclick` to `addEventListener`, why not do the same for `.onmousedown`?